### PR TITLE
fix: #10783 (UI updated on API Creation Form )

### LINKED ISSUE #10783: UI updated on API Creation Form
### Is your proposal related to a problem?

This proposal is basically related to UI updated. While we try to select the days for dropdown it doesn't show properly.
image

### Describe the solution you'd like
I want to update the UI of this form and make a clear dropdown.

### PR DESCRIPTION
**Pull request by hempun10**
## What does this PR do?

Fixes #10783

## Requirement/Documentation

<h1>Before </h1>
image

<h1>After Updated UI</h1>
image

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

## How should this be tested?
Go to Developer Tab in settings and click on create a new API Key.

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
Screenshot 2023-08-16 at 2 28 40 PM

Even with bottom-0 i see everything fine. i can't reproduce the bug

**Comment by hempun10** [comment]
Hey @Udit-takkar  I have updated the changes. Please feel free to review and back me with changes or suggestions.

**Comment by Udit-takkar** [comment]
@hempun10 yeah make the same change in second dialogfooter in the same file

**Comment by hempun10** [comment]
> @hempun10 yeah make the same change in second dialogfooter in the same file

@Udit-takkar  Updated the changes required.

File: packages/ui/components/dialog/Dialog.tsx [hunk 1]
```diff
@@ -153,7 +153,7 @@ export function DialogHeader(props: DialogHeaderProps) {
 
 export function DialogFooter(props: { children: ReactNode; className?: string; showDivider?: boolean }) {
   return (
-    <div className={classNames("bg-default sticky bottom-0", props.className)}>
+    <div className={classNames("bg-default sticky", props.className)}>
```
Comment (Udit-takkar) [inline packages/ui/components/dialog/Dialog.tsx:156]: add bottom-0 like i said in previous comment

Comment (hempun10) [reply packages/ui/components/dialog/Dialog.tsx:156]: Updated the changes

Comment (hempun10) [reply packages/ui/components/dialog/Dialog.tsx]: Added the bottom-0

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**